### DISCUSSION TITLE
How to output tracid to Kong gateway access logs?

### DISCUSSION THREAD

**Discussion by conghuo** [apache/skywalking#12466] [General]
kong version 2.8+
Kong gateway installed Skywalking Kong normally and collected data to OAP
[https://github.com/apache/skywalking-kong](url)
We attempted to obtain tracid through the following log configuration and found that $http_trace_id is an empty value. How should we obtain it?

log_format json '{"access_time":"$time_iso8601",'
                    '"clientip":"$remote_addr",'
                    '"remote_user": "$remote_user", '
                    '"size":$body_bytes_sent,'
                    '"tracid":"$http_trace_id",'}';

**Comment by wu-sheng** [comment]
We don't have update for that. You may need to debug it from nginx context.

**Reply by conghuo** [reply]
OK. Will we consider supporting this feature in the future？

**Reply by wu-sheng** [reply]
There is no consideration, there's no one working on this.

We rely on volunteers working on the codes. There is not an ask-to-do culture.

**Reply by conghuo** [reply]
Thank you!
